.github/workflows/macos.yml

```diff
@@ -79,8 +79,7 @@ jobs:
                        --enable-hardening \
                        --with-curses=ncurses \
                        --prefix=/opt/cobol/gnucobol \
-                       CPPFLAGS="-DREAD_WRITE_NEEDS_FLUSH $CPPFLAGS" \
-                       CFLAGS="-Wno-parentheses-equality $CFLAGS"
+                       CPPFLAGS="-DREAD_WRITE_NEEDS_FLUSH $CPPFLAGS"
```

Comment:
please check if that define is still needed, and if it is add a comment in the build definition for its reason

Reply:
Apparently still required, otherwise we have that failure:
```
1041: LINE SEQUENTIAL COMMIT / ROLLBACK               FAILED (run_file.at:25224)
```
(cf https://github.com/OCamlPro/gnucobol/actions/runs/14515796099/job/40724478508?pr=147)